3rdparty/openssl/define_getpid_noop.h

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+// We force include this header on threads_pthread.c to overwrite getpid(),
+// which isn't supported on Windows.
+#define getpid oe_openssl_get_pid
+
+int oe_openssl_get_pid()
```

Comment:
```suggestion
static OE_INLINE int oe_openssl_get_pid()
```
`static` since this function is not expected to be used elsewhere. `inline` opportunistically.